### ISSUE TITLE
Accessibility: Screen readers doesn't announce any information on the connected nodes or direction of connection

### ISSUE THREAD

**Issue by sulabh-msft** [mermaid-js/mermaid#1013]
**Describe the bug**
Accessibility: Screen readers doesn't announce any information on the connected nodes or direction of connection.

**To Reproduce**
1. Go to [Flowchart](https://mermaidjs.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoiZ3JhcGggVERcbkFbU3RhcnRdIC0tPnxQZXJmb3JtIGFjdGlvbnwgQihTdG9wKVxuXG4iLCJtZXJtYWlkIjp7InRoZW1lIjoiZGVmYXVsdCJ9fQ) 
2. Launch screen reader (e.g narrator or NVDA) and navigate to the right window using caps+arrow key. 

![image](https://user-images.githubusercontent.com/13728069/67091994-c064d700-f1cb-11e9-88b7-d9d28014dae4.png)

**Expected behavior**
Screen reader should announce more detailed information of the image ,there should be an alternative text provided describing the rendered svg.

**Actual:**
On navigating to the image using caps+arrow key , the narrator announces start and stop only , there is no information of it as a connected nodes or the direction of connection.

User Impact:​
The screen reader users will not receive clear context about the image present on the screen.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [Windows]
 - Browser [Edg/Chrome]
 - Version [NVDA/Narrator]

**Additional context**
Similar issue is observed in gantt charts and sequence diagrams

**Comment by jgreywolf** [comment]
I will be closing this issue within 30 days if I do not hear back from anyone that this is still an issue, due to the extreme age with no actitivy
